### DISCUSSION TITLE
top_bar_persistency=audio yet allow it to be toggled on/off

### DISCUSSION THREAD

**Discussion by mrfragger** [tomasklaen/uosc#993] [Ideas]
I always want want it to display for audio files which it does.  Once in a while though might be nice to be able to toggle on/off.  So perhaps some thing like
top_bar_persistency_toggle=audio
which would enable it to by default display.  Right now it's not a big deal to toggle it on/off.  I'm thinking more of the average user though

`e             script-message-to uosc toggle-elements top_bar       #! [Playback] > On/off uosc top bar progress %`

**Comment by hooke007** [comment]
> I'm thinking more of the average user though

Seriously, the average user would not care about it.

**Comment by Sneakpeakcss** [comment]
`e             cycle-values script-opts uosc-top_bar_persistency=audio uosc-top_bar_persistency=""`

**Reply by mrfragger** [reply]
> cycle-values script-opts uosc-top_bar_persistency=audio uosc-top_bar_persistency=""

sweet that does the trick..still need the first one also to toggle for video top bar.  Yea hooke007 maybe you're right but gonna try to distribute this config to a few users who are new to mpv.
